modal/shared/protocol.py

```diff
@@ -85,10 +85,28 @@ class CompletionPayload(BaseModel):
     runner: RunnerConfiguration | None = None
 
 
-class ResponseBody(BaseModel):
-    text: str
+class Usage(BaseModel):
+    """
+    An OpenAI-style usage struct, containing the expected
+    token counts as well as additional data about the GPU
+    usage of the request.
+    """
+
     prompt_tokens: int
     completion_tokens: int
+    total_tokens: int
+
+    # TODO: add these in after deprecating the old fields
+    # duration: float
+    # gpu_type: GPUType
+    # gpu_count: int
```

Comment:
total token count is included in the openai usage payload
https://platform.openai.com/docs/api-reference/chat/object

should we stick to that & include it? feels a bit weird since its a simple sum of existing info lol but included anyway

Reply:
yeah it seems redundant to me. this repo doesn't follow their spec anyway, so I think we could omit it